runelite-client/src/main/java/net/runelite/client/Notifier.java

```diff
@@ -246,6 +246,13 @@ public void processFlash(final Graphics2D graphics)
 			return;
 		}
 
+		if (runeLiteConfig.flashNotification() == FlashNotification.DISABLED)
```

Comment:
You can just move this up to L211 + move up L217

Reply:
Good call, updated